axonserver/src/main/java/io/axoniq/axonserver/message/command/RoutingSelector.java

```diff
@@ -0,0 +1,34 @@
+package io.axoniq.axonserver.message.command;
+
+import java.util.Optional;
+
+/**
+ * Selects the best handler to dispatch the message to, chosen among registered handlers.
+ *
+ * @author Sara Pellegrini
+ * @since 4.3
```

Comment:
Nit: The generic should be described in the javadoc

Reply:
Right